examples/pennylane/7_Pulse_control_OQC_Lucy/7_Pulse_control_OQC_Lucy.ipynb

```diff
@@ -0,0 +1,763 @@
+{
```

Comment:
<p>Great intro!</p><p> </p><p>Under "Introduction", minor nitpicks:</p><ul><li>awk: "Additionally to accessing..." -&gt; "In addition to accessing..."</li><li>awk: "we now have the possibility..." -&gt; "it is now possible to"</li><li>Abbreviations should be spelled out parenthesized at the first instance, and then the abbreviation should be used from that point. Here we have "OQC" in the first section, then "Oxford Quantum Computing (OQC)" in the next section, and then "Oxford Quantum Circuit" spelled out in the third section. Easy to miss in a proof read :) </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Writing something like "Oxford Quantum Computing's (OQC) Lucy" felt awkward in the abstract paragraph, or is there a better way to write it? I just rewrote the sentence in the first paragraph to avoid that:
> We now have the possibility to run
hardware-level circuits combined with standard gates on Lucy by Oxford Quantum Computing (OQC),
accessed in `PennyLane` via `AWS Braket`.